clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2716,6 +2714,8 @@ private StoreResponse getStoreResponse(String storeName, boolean refresh) {
       pushJobSetting.isStoreWriteComputeEnabled = storeResponse.getStore().isWriteComputationEnabled();
       pushJobSetting.isStoreIncrementalPushEnabled = storeResponse.getStore().isIncrementalPushEnabled();
       pushJobSetting.hybridStoreConfig = storeResponse.getStore().getHybridStoreConfig();
+      final boolean isRepush = pushJobSetting.isSourceKafka || pushJobSetting.isSourceETL; // safer to simply allow them
+      pushJobSetting.maxRecordSizeBytes = (isRepush) ? -1 : storeResponse.getStore().getMaxRecordSizeBytes();
```

Comment:
Nit,
Shall we leave an info logs for this behavior?

Reply:
Are you talking about only the `isRepush` code path? I feel like it should be a certain `-1` for that case?

If you're talking about for all cases, then I'm concerned it might be too verbose, but I'm not knowledgeable about this.

Reply:
yes, for the repush path.

Reply:
Okay added the comment:
```java
if (pushJobSetting.maxRecordSizeBytes != -1 && (pushJobSetting.isSourceKafka || pushJobSetting.isSourceETL)) {
  pushJobSetting.maxRecordSizeBytes = -1; // safer to simply allow large records on repush
  final String repushJobType = (pushJobSetting.isSourceKafka) ? "Kafka" : "ETL";
  LOGGER.info("Setting max record size to unlimited for {} repush job", repushJobType);
}
```